maistra/ci/pre-submit.sh

```diff
@@ -4,23 +4,26 @@ set -e
 set -o pipefail
 set -x
 
+export WORKDIR=$(pwd)
```

Comment:
What value do you expect WORKDIR to have? Note that `pwd` returns the location where the user invoked this script, not the location of the script.

Reply:
I expect WORKDIR to have the location where the scripts is invoked, not to use "cabled" absolute paths. This is essential for all the scripts to work both in the builder used in local environment and the builder in prow, as the working directory is different.

Reply:
Move this definition to common.sh as it's used also in post-submit script.

Reply:
Ok

Reply:
forgot to remove it?

Reply:
Yes :-(